crates/matrix-sdk-crypto/src/types/mod.rs

```diff
@@ -233,15 +233,15 @@ impl Serialize for Signatures {
 // likes to base64 encode all byte slices.
 //
 // This ensures that we serialize/deserialize in a Matrix-compatible way.
-fn deserialize_curve_key<'de, D>(de: D) -> Result<Curve25519PublicKey, D::Error>
+pub(crate) fn deserialize_curve_key<'de, D>(de: D) -> Result<Curve25519PublicKey, D::Error>
```

Comment:
Could this also be in vodozemac behind a serde feature?

Reply:
I was pondering moving this into vodozemac as well, but we do want to have the normal `Serialize`/`Deserialize` behavior for `Curve25519PublicKey` in vodozemac itself since the keys get persisted/pickled.

So I left exploring this option for another time.